1.preprocess_data/scripts/1.extract_metadata.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
```

Comment:
Please add a markdown cell describing what this notebook does.

Reply:
Added markdown cell